### PR TITLE
FORMS-12203: Localization of String for wizrad and container

### PR DESCRIPTION
**Pull request by shivangimittal791**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Review by sakshi-arora1 — COMMENTED** [review]
do add a test case for this

**Review by sakshi-arora1 — COMMENTED** [review]
where are the localized values residing?

**Comment by shivangimittal791** [comment]
> where are the localized values residing?

Other components are also using this localization method. Seems this is coming from this dependency which is present in bundles parent pom
<dependency>
   <groupId>org.apache.sling</groupId>
   <artifactId>org.apache.sling.i18n</artifactId>
</dependency>

**Comment by shivangimittal791** [comment]
> do add a test case for this

Added test case

File: ui.tests/test-module/specs/wizard/wizard.authoring.spec.js [hunk 1]
```diff
@@ -155,4 +163,40 @@ describe('Page - Authoring', function () {
...
+            });
+
+            changeLanguage("en");
+            cy.visit('http://localhost:4502/editor.html/content/forms/af/language-test.html');
+            const dataPath = "/content/forms/af/language-test/jcr:content/guideContainer/*",
+                responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+            cy.selectLayer("Edit");
+            cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Wizard Layout", afConstants.components.forms.resourceType.wizard);
+            cy.get('body').click(0, 0);
+            cy.get('[data-path="/content/forms/af/language-test/jcr:content/guideContainer/wizard/*"]').invoke('attr', 'data-text').should('equal', 'Please drag Wizard components here');
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/wizard/wizard.authoring.spec.js:196]: this is only testing for english locale, can you add one more locale.
Rest of PR looks good to me.

Comment (shivangimittal791) [reply ui.tests/test-module/specs/wizard/wizard.authoring.spec.js]: Added test case for another language

File: ui.tests/test-module/specs/wizard/wizard.authoring.spec.js [hunk 2]
```diff
@@ -155,4 +163,51 @@ describe('Page - Authoring', function () {
         });
 
     });
+
+    context('Test Wizard Component String Language', function (){
+        beforeEach( () => {
+            cy.visit('http://localhost:4502/aem/start.html');
+            cy.get('#username').type('admin');
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/wizard/wizard.authoring.spec.js:170]: Why is there a need for this, none of the other spec added this

File: ui.tests/test-module/specs/wizard/wizard.authoring.spec.js [hunk 3]
```diff
@@ -155,4 +163,51 @@ describe('Page - Authoring', function () {
...
+            cy.get('#password').type('admin');
+            cy.get('#submit-button').click();
+            cy.on('uncaught:exception', () => {
+                return false
+            });
+
+            changeLanguage("de");
+            cy.visit('http://localhost:4502/libs/fd/fm/gui/content/forms/af/create.html').then(() => {
+                // Your tests here
+                cy.wait(30000);
```
Comment (dmaurya929) [inline ui.tests/test-module/specs/wizard/wizard.authoring.spec.js:180]: We generally don't use `cy.wait` instead, we wait for a particular event to be triggered. 
If cy.wait is the only option then 30 second delay is huge, it should be improved!

Comment (shivangimittal791) [reply ui.tests/test-module/specs/wizard/wizard.authoring.spec.js]: Updated with cy.wait('@templates',{requestTimeout: 30000});

File: ui.tests/test-module/specs/wizard/wizard.loacle.spec.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+describe('Locale - Authoring Test', function () {
+    const changeLanguage = function (str) {
+        cy.visit('http://localhost:4502/aem/forms.html/content/dam/formsanddocuments');
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userproperties.html"]').click();
+        cy.get('[data-foundation-toggleable-control-src="/mnt/overlay/granite/ui/content/shell/userpreferences.html"]').click();
+        cy.get('[name="language"].coral-Form-field._coral-Dropdown').first().click();
+        cy.get(`coral-selectlist-item[value=${str}]`).click({force: true});
+        cy.get('._coral-Button._coral-Button--cta[trackingelement="accept"]').click();
+    }
+    context('Test Wizard Component String Language', function () {
```
Comment (rismehta) [inline ui.tests/test-module/specs/wizard/wizard.loacle.spec.js:27]: you can use the existing wizard authoring spec, no need to create a new test for this use-case

**Review by rismehta — COMMENTED** [review]
check comments

File: ui.tests/test-module/specs/wizard/wizard.loacle.spec.js (hunk 4 above)
Comment (shivangimittal791) [reply ui.tests/test-module/specs/wizard/wizard.loacle.spec.js:27]: @rismehta I have raised a new PR with all the changes:
https://github.com/adobe/aem-core-forms-components/pull/1152

**Pull request closed without merging**